src/Microsoft.ML.Dnn/DnnUtils.cs

```diff
@@ -266,19 +267,12 @@ internal static DnnModel LoadDnnModel(IHostEnvironment env, string modelPath, bo
             return new DnnModel(env, session, modelPath);
         }
 
-        /// <summary>
-        /// Load TensorFlow model into memory.
-        /// </summary>
-        /// <param name="env">The environment to use.</param>
-        /// <param name="arch">The architecture of the model to load.</param>
-        /// <param name="metaGraph"></param>
-        /// <returns></returns>
-        internal static DnnModel LoadDnnModel(IHostEnvironment env, ImageClassificationEstimator.Architecture arch, bool metaGraph = false)
+        internal static DnnModel LoadDnnModel(IHostEnvironment env, Architecture arch, bool metaGraph = false)
         {
-            var modelPath = ImageClassificationEstimator.ModelLocation[arch];
+            var modelPath = ModelLocation[arch];
             if (!File.Exists(modelPath))
             {
-                if (arch == ImageClassificationEstimator.Architecture.InceptionV3)
+                if (arch == Architecture.InceptionV3)
                 {
                     var baseGitPath = @"https://raw.githubusercontent.com/SciSharp/TensorFlow.NET/master/graph/InceptionV3.meta";
```

Comment:
Without pinning to a specific git commit, the referenced file can be move/changed causing a breakage.  
```suggestion
                    var baseGitPath = @"https://raw.githubusercontent.com/SciSharp/TensorFlow.NET/547c4e6bf48fceb8266d0cb88d3a13ec4171606a/graph/InceptionV3.meta";
```

Reply:
I have removed this change as part of this PR but in this case we plan to move this to Azure Blob

---
In reply to: [338996402](https://github.com/dotnet/machinelearning/pull/4384#discussion_r338996402) [](ancestors = 338996402)